vm/actor/src/lib.rs

```diff
@@ -53,6 +53,16 @@ where
     Map::<_, V>::new_with_bit_width(store, HAMT_BIT_WIDTH)
 }
 
+/// Create a hamt with a custom bitwidth.
+#[inline]
+pub fn make_map_with_bitwidth<BS, V>(store: &'_ BS, bitwidth: u32) -> Map<'_, BS, V>
```

Comment:
small point, if both of these always requires a bit width, perhaps you don't need the `_with_bitwidth` suffix. Just an opinion though, this is totally fine.

Reply:
hah, makes sense. im gunna completely remove `make_map` and replace it with this.

Reply:
yeah can definitely do later, I'm just suggesting more as something to move toward